modules/libcom/RTEMS/posix/rtems_config.c

```diff
@@ -148,14 +149,15 @@ extern void *POSIX_Init(void *argument);
 #define CONFIGURE_SHELL_COMMAND_SHUTDOWN
 
 #include <rtems/shellconfig.h>
+
 #define RTEMS_BSD_CONFIG_BSP_CONFIG
 #define RTEMS_BSD_CONFIG_SERVICE_TELNETD
 #define RTEMS_BSD_CONFIG_TELNETD_STACK_SIZE (16 * 1024)
 #define RTEMS_BSD_CONFIG_SERVICE_FTPD
 #define RTEMS_BSD_CONFIG_FIREWALL_PF
-#else
+#else // __RTEMS_MAJOR__ > 4
 #include <rtems/shellconfig.h>
```

Comment:
Does some additional header need to be included for shell commands coming from libbsd?